xdg/openshot-qt.appdata.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+	<id>openshot-qt.desktop</id>
+	<metadata_license>CC0-1.0</metadata_license>
+	<project_license>GPL-3.0+</project_license>
+	<name>OpenShot</name>
+	<url type="homepage">https://www.openshot.org</url>
+	<summary>Video Editor</summary>
+	<description>
+		<p>We designed OpenShot Video Editor to be an easy to use, quick to learn, and surprisingly powerful video editor.</p>
+	</description>
+	<screenshots>
+		<screenshot type="default">
+			<image>https://www.openshot.org/static/img/gallery/ui-example.jpg</image>
+		</screenshot>
+		<screenshot>
+			<image>https://www.openshot.org/static/img/gallery/title-editor.jpg</image>
+		</screenshot>
+	</screenshots>
+	<releases>
+		<release version="2.4.1" date="2017-11-12"/>
+  	</releases>
+	<content_rating type="oars-1.0">
+		<content_attribute id="violence-cartoon">none</content_attribute>
+		<content_attribute id="violence-fantasy">none</content_attribute>
+		<content_attribute id="violence-realistic">none</content_attribute>
+		<content_attribute id="violence-bloodshed">none</content_attribute>
+		<content_attribute id="violence-sexual">none</content_attribute>
+		<content_attribute id="drugs-alcohol">none</content_attribute>
+		<content_attribute id="drugs-narcotics">none</content_attribute>
+		<content_attribute id="drugs-tobacco">none</content_attribute>
+		<content_attribute id="sex-nudity">none</content_attribute>
+		<content_attribute id="sex-themes">none</content_attribute>
+		<content_attribute id="language-profanity">none</content_attribute>
+		<content_attribute id="language-humor">none</content_attribute>
+		<content_attribute id="language-discrimination">none</content_attribute>
+		<content_attribute id="social-chat">none</content_attribute>
+		<content_attribute id="social-info">none</content_attribute>
```

Comment:
Based on OpenShot's ability to (optionally, but on by default) send tracking metrics back to the "mothership", I believe this should be at _least_ "moderate", possibly even "intense". (The telemetry doesn't _contain_ any personally-identifiable information, but it does include a unique identifier that ties the data to a specific OpenShot instance installed on a specific system, which could theoretically be associated with a particular user. I'm not clear on whether that's enough to raise the rating to "intense".)

Reply:
(The "This" in my previous comment being OpenShot's `social-info` content rating, just to be clear.)